src/lib/snack-bar/snack-bar.md

```diff
@@ -72,3 +72,7 @@ export class MessageArchivedComponent {
   constructor(@Inject(MD_SNACK_BAR_DATA) public data: any) { }
 }
 ```
+### Accessibility
+A Snackbar automatically goes away after a short time, so you can't count on the user seeing the
```

Comment:
Nit: To be consistent with the rest of the doc, use `snack-bar` instead of `Snackbar` ?

Reply:
Changed to `snack-bar`. Thanks for review.